soc/nxp/kinetis/k6x/Kconfig.defconfig

```diff
@@ -1,5 +1,6 @@
-# Kinetis K6x series configuration options
+# Kinetis K6x series default configurations
 
+# Copyright 2024 NXP
 # Copyright (c) 2014-2016 Wind River Systems, Inc.
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
Based on [this](https://github.com/zephyrproject-rtos/zephyr/pull/68778/commits/6d7b5706751f7367c1b14eab82d2a442bffa5b1d#r1483946471) comment, the SOC_SERIES from this defconfig should be moved to Kconfig. soc

LE: the same for k8x, ke1xf, kv5x